### PR TITLE
Improvement: documentation

### PR DESCRIPTION
**Pull request by Zorin95670**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,16 +28,67 @@ activated or deactivated for users, groups and roles.
...
+This role is designed for users who primarily work on projects, diagrams, and components.
+
+They have all permissions related to these areas, enabling them to create, modify, and delete project-related content.
+
+However, their access is restricted in administrative functions outside their scope.
+
+### Manage user groups
+
+
+The application offers comprehensive functionality for managing groups and the users within these groups, enabling precise control over access and roles assigned to each member.
```
Comment (justinehell) [inline README.md:87]: I would suggest using another word than `comprehensive` (false friend in french), better use ` complete` or `full`.

Also, maybe add example of role ?

**Review by justinehell — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -28,16 +28,67 @@ activated or deactivated for users, groups and roles.
...
+- `DEVELOPER`
+
+This role is designed for users who primarily work on projects, diagrams, and components.
+
+They have all permissions related to these areas, enabling them to create, modify, and delete project-related content.
+
+However, their access is restricted in administrative functions outside their scope.
+
+### Manage user groups
+
```
[the remark this answers is not in the document]
Comment (Zorin95670) [reply README.md:85]: done

File: README.md (hunk 1 above)
Comment (Zorin95670) [reply README.md:87]: done

File: src/main/resources/db/migration/V1_7_0__new_table_library_templates.sql [hunk 3]
```diff
@@ -21,9 +21,9 @@ COMMENT ON TYPE   library_template_type               IS 'An ENUM type represent
 COMMENT ON TABLE  library_templates                   IS 'This table stores information about various libraries used within the diagram system.';
 COMMENT ON COLUMN library_templates.lit_id            IS 'Primary key, serial.';
 COMMENT ON COLUMN library_templates.lib_id            IS 'References the lib_id in the libraries table. This optional field links the template to a specific library.';
-COMMENT ON COLUMN library_templates.name              IS 'The name of the library.';
+COMMENT ON COLUMN library_templates.name              IS 'The name of the library template.';
 COMMENT ON COLUMN library_templates.type              IS 'Specifies the type of entity for which the template is being set, based on the library_template_type ENUM.';
-COMMENT ON COLUMN library_templates.description       IS 'A text description of the library.';
-COMMENT ON COLUMN library_templates.documentation_url IS 'The documentation URL of the template.';
+COMMENT ON COLUMN library_templates.description       IS 'A text description of the library library.';
```
[the remark this answers is not in the document]
Comment (Zorin95670) [reply src/main/resources/db/migration/V1_7_0__new_table_library_templates.sql]: done

**Review by justinehell — APPROVED** [review]

**Pull request merged**